src/Microsoft.Health.Fhir.Core/Features/Conformance/Serialization/EnumLiteralJsonConverter.cs

```diff
@@ -21,7 +21,17 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
             if (value is Enum obj)
             {
-                serializer.Serialize(writer, obj.GetLiteral());
+                string enumString;
```

Comment:
Would be helpful to add a comment on why this change is being made. Will make it easier in the future if we have to deal with something similar or have to modify this behavior again.